tests/php/Forms/NumericFieldTest.php

```diff
@@ -127,6 +127,19 @@ public function testNumberTypeOnInputHtml()
         $this->assertContains('type="text"', $html, 'number type not set');
     }
 
+    public function testNullSet() {
+        $field = new NumericField('Number');
+        $field->setValue('');
+        $this->assertNull($field->Value());
```

Comment:
I'll leave this for @pitchandtone to tidy up. Remember to pull the commit locally that I've pushed up...